packages/octocode-mcp/src/prompts.ts

```diff
@@ -5,6 +5,7 @@ import { PROMPT_SYSTEM_PROMPT } from './systemPrompts';
 // Define prompt names as constants (following your project's pattern)
 export const PROMPT_NAMES = {
   RESEARCH: 'research',
+  KUDOS: 'kudos',
```

Comment:
Maybe worth to change to /sources

Reply:
But `kudos` or `thanks` sound more like appreciation, while `sources` comes off as more formal. I just wanted to keep it friendly, but the final choice is yours :)